docs/benefits_of_contextualized_ML.ipynb

```diff
@@ -0,0 +1,804 @@
+{
```

Comment:
<p>Hmm these are quite difficult to read, is it possible to use a cmin/max just for the 3 that are plotted so edges are more obvious?</p><p> </p><p>Another colorscale might help. Maybe the coolwarm colorscale would look nice? Green as a background color isn't so intuitive to me.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cnellington/Contextualized/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>